spec/stacktrace_spec.rb

```diff
@@ -12,8 +12,9 @@
 
     expect(Bugsnag).to have_sent_notification{ |payload, headers|
       exception = get_exception_from_payload(payload)
-      starting_line = __LINE__ - 10
-      expect(exception["stacktrace"][1]["code"]).to eq({
+      starting_frame = get_project_frame(exception["stacktrace"], 1)
```

Comment:
Instead of using this function, use a function that explicitly raises and catches an exception to notify - rather than notifying an exception with no stack.